response_operations_ui/exceptions/error_codes.py

```diff
@@ -14,15 +14,19 @@ class ErrorCode(Enum):
     API_OIDC_CREDENTIALS_ERROR = "E0006"
     API_TIMEOUT_ERROR = "E0007"
 
+    # HTTP Error Codes
+    NO_RESULTS_FOUND = "404"
+
 
 def get_error_code_message(error_code):
     error_code_messages = {
-        ErrorCode.API_CONNECTION_ERROR: "An error occurred trying to connect to the service",
+        ErrorCode.API_CONNECTION_ERROR: "Unable to connect to CIR",
```

Comment:
This shouldn't be specific to the CIR. It's intended to be used for any service we make requests to.

Reply:
That's very true. Updated.